src/CategoryCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Regarding your question about where should you put this component. I agree, we should definitely discuss the structure. You can start a discussion about it on slack, so all the girls can easily get involved :)